test/e2e/test_auditlogs.go

```diff
@@ -48,15 +43,15 @@ func CloudAuditLogsSourceWithTestImpl(t *testing.T, authConfig lib.AuthConfig) {
 	defer lib.TearDown(client)
 
 	// Create a target Job to receive the events.
-	lib.MakeAuditLogsJobOrDie(client, methodName, project, resourceName, serviceName, targetName)
+	lib.MakeAuditLogsJobOrDie(client, lib.MethodName, project, resourceName, lib.ServiceName, targetName)
```

Comment:
Should these still be arguments now that that constants have moved?

Reply:
I think it will be better if we still make these to be arguments so that callers can customize the AuditLogsJobs.